### PR TITLE
DF-1565 new stes-related views created, aux_code table created and fi…

### PR DESCRIPTION
**Pull request by dvoracek-martin**
…le for data import to that table has been added

### CODE REVIEW

File: alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20191007120000_modify_datatype_by_fyn_fields.xml [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+           WHEN BILDUNGSSTUFE.CODE BETWEEN '150' AND '179'
+               THEN
+               CAST ('ISCED5' AS VARCHAR (6))
+           WHEN BILDUNGSSTUFE.CODE = '180'
+               THEN
+               CAST ('ISCED6' AS VARCHAR (6))
+           END
+           AS AUSBILDUNG_CODE,
+       CAST (BILDUNGSSTUFE.CODE AS NUMERIC) AUSBILDUNGS_NIVEAU,
+       cast (case  A.FYN_AUSGEUEBT   when 'Y' then  1   when 'N' then  0 end as NUMERIC) AUSGEUEBT_B,
```
Comment (pavax) [inline alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20191007120000_modify_datatype_by_fyn_fields.xml]: @dvoracek-martin  I thought we should also consider the case where lambda sends us '-'. (Even though we've only observed this behaviour so far in one-column). What is your opinion?

Comment (dvoracek-martin) [reply alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20191007120000_modify_datatype_by_fyn_fields.xml]: True. Transforming hyphen to 0 was missing and the hyphen was saved as it was instead. Fixed.

Comment (dvoracek-martin) [reply alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20191007120000_modify_datatype_by_fyn_fields.xml]: You're correct that it should have a default value 0 for anything else than Y or N. But when I'm thinking about it now, we've created a catdog - on some fieleds we do this transformation to 1 and 0, somewhere we keep Y/N. I assume we should transform the same way to 1 and 0 in all the fields. This needs to be fixed yet.

Comment (dvoracek-martin) [reply alv-lamda-csv-importer/src/main/resources/liquibase/changelog/20191007120000_modify_datatype_by_fyn_fields.xml]: @pavax  You're correct, the default value was missing. Fixed

**Pull request merged**